### PR TITLE
docs(changeset): feat: add breakpoints / viewports

### PR DESCRIPTION
**Pull request by larsrickert**
Add Onyx breakpoints to Storybook so the user can easily select one of them to see how the components behave on different screen sizes.

### CODE REVIEW

**Review by jannick-ux — DISMISSED** [review]

File: packages/storybook-utils/src/theme.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/** All available Storybook breakpoints / viewports supported by Onyx. */
+export const ONYX_BREAKPOINTS = {
+  "2xsmall": {
```
Comment (BoppLi) [inline packages/storybook-utils/src/theme.ts:3]: [optional]
```suggestion
  xsmall2 : {
```

would UX also be ok with that? then we wouldn't have to wrap them in "" ^^

**Review by BoppLi — DISMISSED** [review]

File: packages/storybook-utils/src/theme.ts (hunk 1 above)
Comment (BoppLi) [reply packages/storybook-utils/src/theme.ts:3]: or xxsmall

Comment (larsrickert) [reply packages/storybook-utils/src/theme.ts:3]: The breakpoint names are defined by @jannick-ux in Figma: https://www.figma.com/file/YfEUBOHk4J4nYrk04geswG/onyx---Conception-%26-DEV-Exchange?type=design&node-id=14-698&mode=dev

Comment (jannick-ux) [reply packages/storybook-utils/src/theme.ts:3]: The names of the breakpoints have never been used with "" by myself. So having it without those signs is intended.

**Review by BoppLi — APPROVED** [review]

**Pull request merged**